### ISSUE TITLE
Option for number of spaces as indentation

### ISSUE THREAD

**Issue by arvinsim** [yosssi/ace#47]
Current indentation rule is 2 spaces. 
It would be better if there is an option to set this to to other values like 4 spaces.

**Comment by yosssi** [comment]
Thanks for your comment. I'll think of adding this option.

**Comment by TheHippo** [comment]
It looks like I am in the wrong place, but since we are at this topic: How about allowing <kbd>TAB</kbd> for indention?

**Comment by yosssi** [comment]
Thanks for your comment. I think it would be nice if we could add an option that specifies the indent rule (2 spaces, 4 spaces or tab) to the [Options](https://godoc.org/github.com/yosssi/ace#Options) struct.

**Comment by TheHippo** [comment]
I'll try to look into this.

The fix should be applied somewhere around [here in line.go](/yosssi/ace/blob/1f82044938a7180f6fb6bbb3a29688d1e6dbe74b/line.go#L106-L117)

**Comment by jf** [comment]
YES. Tabs, pls!!! :} Just like go, and just like a lot of sane code... :}
